notebooks/285-surya-line-level-text-detection/README.md

```diff
@@ -0,0 +1,27 @@
+# Line-level text detection with Surya
+
+[![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/openvinotoolkit/openvino_notebooks/blob/main/notebooks/285-surya-line-level-text-detection/285-surya-line-level-text-detection.ipynb)
+
+In this tutorial we will perform line-level text detection using [Surya](https://github.com/VikParuchuri/surya) toolkit and OpenVINO.
+
+![line-level text detection](https://github.com/VikParuchuri/surya/blob/master/static/images/excerpt.png?raw=true)
+
+[**image source*](https://github.com/VikParuchuri/surya)
+
+
+Model used for line-level text detection based on [Segformer](https://arxiv.org/pdf/2105.15203.pdf). It has the following features:
+* It is specialized for document OCR. It will likely not work on photos or other images.
+* It is for printed text, not handwriting.
+* The model has trained itself to ignore advertisements.
+* Languages with very different character sets may not work well.
+
+#### Table of contents:
+1. Fetch test image.
+1. Run PyTorch inference.
```

Comment:
Change the nubmering

Reply:
This is markdown feature that numbers don't have to be in numerical order: https://www.markdownguide.org/basic-syntax/#ordered-lists. The markdown application will render lists in numerical order anyway.
Or maybe there is another reason for ordering?